backend/src/test/java/gov/cdc/usds/simplereport/service/PersonServiceTest.java

```diff
@@ -100,7 +113,8 @@ void roundTrip() {
         null,
         false,
         false);
-    List<Person> all = _service.getAllPatients(PATIENT_PAGEOFFSET, PATIENT_PAGESIZE);
```

Comment:
These verbose names are nice. But they are ONLY used from test code. All the production code uses `_service.getPatients()` with different parameters to change the behavior.

By removing the names, we've reduced a bunch of boilerplate code and made tests calls closer to production calls.